apm-sniffer/apm-sdk-plugin/feign-default-http-9.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/feign/http/v9/DefaultHttpClientInterceptor.java

```diff
@@ -20,16 +20,13 @@
 
 import feign.Request;
 import feign.Response;
+
 import java.lang.reflect.Field;
 import java.lang.reflect.Method;
 import java.lang.reflect.Modifier;
 import java.net.URL;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.LinkedHashMap;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
does not import *

Reply:
This is not required. If we want to fo this, we need to add style check. It is hard to do manually. @aderm

Reply:
Resolved